test/credential.policies.e2e-spec.ts

```diff
@@ -216,7 +219,20 @@ describe('Issue and Prove credentials using policies (e2e)', () => {
                     throw e;
                 }
             });
-    }, 30000);
```

Comment:
these timeouts we might want to keep.   Both Jeff and I seem to have docker performance issues.  We can add them back later if we need them.....